tests/test_database.py

```
@@ -0,0 +1,158 @@
+            next(session_gen)
+        except StopIteration:
+            pass
+
+    def test_get_session_closes_session(self):
+        """Test get_session properly closes the session"""
+        from fastapi_toolbox.database import init_database, get_session
+
+        init_database(database_url="sqlite:///:memory:")
+
+        session_gen = get_session()
+        session = next(session_gen)
```

Ruff (0.14.10)

Local variable `session` is assigned to but never used

Remove assignment to unused variable `session`

(F841)